.travis.yml

```diff
@@ -3,9 +3,9 @@ language: ruby
 cache: bundler
 
 rvm:
-  # - 2.0.0
-  # - 2.1.1
-  - 2.1.2
+  - 2.0.0
```

Comment:
Not required for this checkin; but if you just say `2.0` and `2.1`, I think Travis will pick the latest.